src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -248,7 +248,7 @@ protected override ImmutableArray<PendingBranch> Scan(ref bool badRegion)
                 _returnTypesOpt.Clear();
             }
             this.Diagnostics.Clear();
-            ParameterSymbol methodThisParameter = MethodThisParameter;
```

Comment:
>MethodThisParameter [](start = 50, length = 19)

Should the implementation of `MethodThisParameter` be changed to use `_symbol.TryGetThisParameter(...)`?